Content.Server/Temperature/Systems/TemperatureSystem.cs

```diff
@@ -15,6 +15,8 @@
 using Robust.Shared.Prototypes;
 using Robust.Shared.Physics.Events;
 using Content.Shared.Projectiles;
```

Comment:
Unused using directive
```suggestion
using Robust.Shared.Prototypes;
using Content.Shared.Projectiles;
```